gulpfile.js/build.js

```diff
@@ -529,18 +529,22 @@ function collectStatics(done) {
  * @return {undefined}
  */
 function persistBuildInfo(done) {
-  const buildInfo = {
-    'number': travis.build.number || null,
-    'at': new Date(),
-    'by': git.user,
-    'environment': config.environment,
-    'commit': {
-      'sha': git.version,
-      'message': git.message,
-    },
-  };
-
-  fs.writeFile(project.paths.BUILD_INFO, yaml.safeDump(buildInfo), done);
+  Promise.all([git.user(), git.version(), git.message()])
```

Comment:
we are using async/await elsewhere in the file - any reason you aren't here?

Reply:
Didn't want to change the the function declaration ;)
I'll give it a try